clang/Type.d

```diff
@@ -130,13 +169,10 @@ struct Type
         return ArrayType(this);
     }
 
-    @property bool isArray ()
+    @property string toString() const
     {
-        return
-            kind == CXTypeKind.CXType_ConstantArray ||
-            kind == CXTypeKind.CXType_IncompleteArray ||
-            kind == CXTypeKind.CXType_VariableArray ||
-            kind == CXTypeKind.CXType_DependentSizedArray;
+        import std.format: format;
+        return format("Type(kind = %s, spelling = %s, isConst = %s)", kind, spelling, isConst);
     }
 }
 
```

Comment:
Ok, I'll try to remember about that. I've just grouped some kinds of methods together.